src/ott/initializers/linear/initializers.py

```diff
@@ -32,22 +34,29 @@ class SinkhornInitializer(abc.ABC):
 
   @abc.abstractmethod
   def init_dual_a(
-      self, ot_prob: 'linear_problem.LinearProblem', lse_mode: bool
+      self,
+      ot_prob: linear_problem.LinearProblem,
+      lse_mode: bool,
+      rng: Optional[jax.random.PRNGKeyArray] = jax.random.PRNGKey(0)
```

Comment:
This is no longer `Optional`, please remove (also in other places.

Reply:
What's optional for? So for this I agree it's needed but for other initialisers like Gaussian initialiser, it should be optional right given it's not used?

Reply:
`Optional[T]` is an alias for `Union[T, None]`; just means the value can be `None`. In our case it cannot be `None`, therefore it should be removed.